### ISSUE TITLE
Service landing page, top tasks: difference between blue and green links unclear

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_services#281]
On a service landing page we can add Top Tasks

These links can be either "Action" or "Information" links. 

Assigning one or the other makes a the link blue or green in the base theme. 

I have a number of UX / accessibility concerns about this. 

1. Non-visual users are not given clues to the different type of links, unless the link text is clear
2. People who can't identify colour difference will not know the difference between the link types, unless the link text is clear
1. for users who _can_ identify colour difference, is the meaning of these colours clear? 

Possible options:

1. Find out if end users understand the difference between blue and green (information or action) links is clear. 
3. Add some kind of visual label or icon (with visually hidden alternative text)
4. Add help text on how to write Action or Information link text

https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html

**Comment by msayoung** [comment]
@willguv - might be one for the Refresh mission? 

I'm not sure how many councils are actually using this feature in the wild. 

Has this come up in any of the consultations so far?

**Comment by willguv** [comment]
@msayoung not sure about use, but we found confusing at Brighton & Hove when it was originally suggested

Presumably removing this feature would resolve the issue

I'll add this to the services page refresh work

**Comment by willguv** [comment]
Being discussed here https://localgovdrupal.slack.com/archives/C020SJRQX2M/p1752580457365949

**Comment by willguv** [comment]
@markconroy let's chat about what to do with this

We're thinking remove the drop-down and go for one color

**Comment by markconroy** [comment]
Moved to Drupal.org

https://www.drupal.org/project/localgov_services/issues/3576493
